### ISSUE TITLE
部署服务端错误

### ISSUE THREAD

**Issue by DeanZeng** [XX-net/XX-Net#1154]
日志
Oct 05 17:56:46 - upload fail: HTTP Error 404: 

Oct 05 17:56:46 - Retry 1 time...

Oct 05 17:56:46 - Application: dean-1065
Oct 05 17:56:46 - Host: appengine.google.com
Oct 05 17:56:46 - Rolling back the update.
Oct 05 17:57:06 - upload fail: The read operation timed out

**Comment by simonasd** [comment]
未能与GAE服务器建立连接。你开启了IPV6，请确认你的IPV6是可用状态

**Comment by sg157** [comment]
能不能说详细一点到底该怎么做呀！

**Comment by simonasd** [comment]
@sg157 你指的是哪部分详细一点？如果是说验证IPV6是否可用，则访问 http://www.kame.net/ ，当IPV6可用时，图片中的乌龟是会动的。

至于如何排查其他的连接问题，就事论事。我只是看到楼主的状态信息显示未能建立GAE连接，同时开启了IPV6。所以只能先让楼主确定一下IPV6没有问题，再想办法进一步排查。

**Comment by sg157** [comment]
不是ipv6的问题吧。你应该去看一下Gmail应该收到一个邮件说的是，不安全的应用是否开启的问题吧。我的问题跟他的问题一样已经解决了

**Comment by simonasd** [comment]
@sg157 但是楼主的连接数是0

**Comment by sg157** [comment]
@simonasd。这个，我当时只是大概看了一下内容，没看连接数。

**Comment by simonasd** [comment]
@sg157 连接数是0的话，就说明和服务端还没有建立连接，因此密码问题、弱安全应用问题还涉及不到。

**Comment by sg157** [comment]
好吧(∩_∩)！我不懂原理，楼主的问题不知道解决了没有呢。你是说的这个的链接数吗？connected link(帮助) 0,0

**Comment by simonasd** [comment]
@sg157 是的。這算一個比較重要的指標吧，你可以看一下wiki中的[解釋](https://github.com/XX-net/XX-Net/wiki/GoAgent-Connection-status)。

**Comment by sg157** [comment]
@simonasd 好吧。谢谢啦！

**Comment by simonasd** [comment]
@sg157 刚看到反馈，连接数为0的时候，有时也能代理上网。有点疑惑，需要再搞搞清楚。

@DeanZeng 楼主问题解决了吗？我在[issue 1151](https://github.com/XX-net/XX-Net/issues/1151) 中看到有网友说：    
在“收到的邮件里有链接或按钮，点击，打开的新页面里有解锁的（不一定是这名，但就是这意思）然后就能部署了”     
不知道你的情况是什么样的？
